components/FavoritesCard.tsx

```diff
@@ -0,0 +1,85 @@
+import { Text } from "./Themed";
+import { StyleSheet, View, Image } from "react-native";
+import { useSelector } from "react-redux";
+import { currentReduxUser } from "../redux/signin";
+import { useEffect, useState } from "react";
+import { getOneDocById, getAllDocsInCollection } from "../helper";
+
+export const FavoritesCard = () => {
+  const [urls, setUrls] = useState<string[]>([]);
+  const user = useSelector(currentReduxUser);
+  const favoritesArray: any = [];
+  let photoURLS: string[] = [];
+
+  useEffect(() => {
+    getLiked();
+    compareLikedIds();
+    imagesLoaded();
+  }, []);
+
+  const getLiked = async () => {
+    try {
+      const favorites = await getOneDocById("users", user?.id);
```

Comment:
This will create a problem if you visit another users profile. 'user' will always be the current logged in user so you will only get your own. To avoid this you need to change from _getOneDocById("users", user?.id)_  to _getOneDocById("users", *route.params.id*);_